### PR TITLE
[zh] Sync cluster-level-pss.

### PR DESCRIPTION
**Pull request by 1000Delta**
Add code sample that will be applied to the cluster.

Based on #44137

### CODE REVIEW

**Comment by 1000Delta** [comment]
See [Preview](https://deploy-preview-44211--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/security/cluster-level-pss/#set-modes-versions-and-standards) at step 7.

File: content/zh-cn/docs/tutorials/security/cluster-level-pss.md [hunk 1]
```diff
@@ -434,6 +435,9 @@ following:
 -->
 7. 在 default 名字空间下创建一个 Pod：
 
+
+    {{% code_sample file="security/example-baseline-pod.yaml" %}}
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/security/cluster-level-pss.md:439]:
```suggestion
   {{% code_sample file="security/example-baseline-pod.yaml" %}}
```
The en indentation is not very correct. Normally, it's 3 spaces for 1-2-3 list

Comment (1000Delta) [reply content/zh-cn/docs/tutorials/security/cluster-level-pss.md:439]: Do we have some guideline to determine this?

In [Style Guide - Lists](https://kubernetes.io/zh-cn/docs/contribute/style/style-guide/#lists) it said should use 4 spaces, but in fact are 2 spaces for unordered list and 3 spaces for ordered list?

Comment (windsonsea) [reply content/zh-cn/docs/tutorials/security/cluster-level-pss.md]: style guide 说的是嵌套的2级无序列表吧：

- list 0
  - list 1
    - list 2

**Review by windsonsea — COMMENTED** [review]
/approve

File: content/zh-cn/docs/tutorials/security/cluster-level-pss.md (hunk 1 above)
Comment (1000Delta) [reply content/zh-cn/docs/tutorials/security/cluster-level-pss.md]: 这里应该对应的是，最后一点中，列表包含多个段落的情况？

**Review by chenrui333 — APPROVED** [review]
/lgtm

**Pull request merged**